core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.scss

```diff
@@ -3,7 +3,7 @@
 :host {
     display: grid;
     flex-direction: column;
-    grid-template-rows: 48px 1fr;
+    grid-template-rows: 100px 1fr;
```

Comment:
why?

Reply:
The toolbar was breaking into the iframe

Reply:
Looks like a `fit-content` to me but I have to run it.